config/samples/replication.storage_v1alpha1_volumegroupreplicationcontent.yaml

```diff
@@ -9,4 +9,11 @@ metadata:
     app.kubernetes.io/created-by: kubernetes-csi-addons
   name: volumegroupreplicationcontent-sample
 spec:
-  # TODO(user): Add fields here
+  volumeGroupReplicationRef:
+    name: volumegroupreplication
+    namespace: default
+  volumeGroupReplicationHandle: replicationHandle
+  provisioner: driver.csi.com
+  volumeGroupReplicationClassName: volumegroupreplicationClass
```

Comment:
` volumeGroupReplicationClassName` -> ` volumeGroupReplicationClass` , it should be same as in the above yaml?

Reply:
it should be the className as defined in the api